array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,14 @@
+/* exported equal */
+function equal(first, second) {
+  var combinedArr = [];
+  for (var i = 0; i < first.length; i++) {
+    if (first[i] === second[i]) {
+      combinedArr.push(first[i]);
+    }
+  }
+  var isSameLength = second.length === combinedArr.length;
+  if (isSameLength) {
+    return true;
+  }
```

Comment:
After making the above changes, if the code makes it to here the arrays are equal so you can just return true.